### PR TITLE
Fixed ACLAction Travis failure

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

This was happening due to trying to use $this in static methods. While here I also cleaned up some inspections.

## How To Test This

See Travis.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: tests/api/V8/GetModulesMetaCest.php [hunk 1]
```diff
@@ -36,9 +36,9 @@ public function shouldWork(ApiTester $I, Example $example)
             [
                 'data' =>
                     [
-                        'Accounts' => [
-                            'label' => 'Accounts',
-                            'access' => ['access']
+                        'type' => 'modules',
+                        'attributes' => [
```
Comment (Dillon-Brown) [inline tests/api/V8/GetModulesMetaCest.php]: Just to note, this isn't actually a fix. Just wanted to see if Travis would pass :).

**Comment by LeoZandvliet** [comment]
Sorry to bother you, but will this PR also target the Travis errors appearing on the hotfix 7.10.x branch?
For example:

- https://travis-ci.org/salesagility/SuiteCRM/builds/601949651
- https://travis-ci.org/salesagility/SuiteCRM/builds/601969546
- https://travis-ci.org/salesagility/SuiteCRM/builds/601787966

**Comment by Mac-Rae** [comment]
Assessed :+1: LGTM

**Comment by Dillon-Brown** [comment]
Hi @LeoZandvliet, that should be the hotfix-7.10.x branch passing now. I've gone through a few of the recent PRs and merged hotfix-7.10.x into them too so we should be good. Thanks!

**Pull request merged**